src/ocean/util/container/pool/FreeList.d

```diff
@@ -312,6 +317,36 @@ version ( UnitTest )
         }
     }
 
+    /***************************************************************************
+
+        Buffer free list tester.
+
+    ***************************************************************************/
+
+    alias FreeList!(Buffer!(void)) BufferFreeList;
+    class BufferFreeListTester : FreeListTester!(BufferFreeList.ItemType)
```

Comment:
It looks like those types, albeit hidden being `version (UnitTest)`, are public ?

Reply:
Good catch. Won't make a difference for D1 but should fix anyway.